cypress/features/regression/buttonToggle.feature

```diff
@@ -79,8 +76,19 @@ Feature: Button Toggle component
     Then Button Toggle is grouped
 
   @positive
-  Scenario: Enable and disable Button Toggle grouped property
-    When I check grouped checkbox
-      And I uncheck grouped checkbox
+  Scenario: Disable Button Toggle grouped property
+    Given I check grouped checkbox
+    When I uncheck grouped checkbox
       And I enable Button component
     Then Button Toggle is not grouped
+
+  @positive
+  Scenario Outline: Verify the onChange event for Button Toggle <button>
+    Given clear all actions in Actions Tab
+    When I click on Button Toggle <button>
+    Then <action> action was called in Actions Tab
+    Examples:
+      | button | action   |
+      | 0      | onChange |
+      | 1      | onChange |
+      | 2      | onChange |
```

Comment:
we should remove action from Examples and put it directly in `Then` step if it has no difference :)

Reply:
Amended 👍